OsmAnd/res/values/strings.xml

```diff
@@ -11,21 +11,21 @@
 	Thx - Hardy
 -->
 
-    <string name="switch_mode_by_tapping_on_widget">Switch the mode by tapping on the widget.</string>
-    <string name="map_widget_sunset_desc">Shows the time of the next sunset or the time left until sunset.</string>
-    <string name="map_widget_sunrise_desc">Shows the time of the next sunrise or the time left until sunrise.</string>
+    <string name="switch_mode_by_tapping_on_widget">Switch mode by tapping the widget.</string>
+    <string name="map_widget_sunset_desc">Shows time of, or time left until next sunset.</string>
+    <string name="map_widget_sunrise_desc">Shows time of, or time left until next sunrise.</string>
     <string name="shared_string_next_sunset">Next sunset</string>
     <string name="shared_string_next_sunrise">Next sunrise</string>
     <string name="shared_string_time_left">Time left</string>
     <string name="shared_string_sunset">Sunset</string>
     <string name="shared_string_sunrise">Sunrise</string>
-    <string name="map_widget_group_sunrise_sunset_desc">These widgets show information about the next sunrise and sunset.</string>
+    <string name="map_widget_group_sunrise_sunset_desc">Shows info about next sunrise and sunset.</string>
     <string name="map_widget_group_sunrise_sunset">Sunrise, sunset</string>
-    <string name="rendering_attr_depthContourColorScheme_name">Line color scheme</string>
+    <string name="rendering_attr_depthContourColorScheme_name">Line color-scheme</string>
     <string name="rendering_attr_depthContourWidth_name">Line width</string>
-    <string name="depth_contour_lines">Depth contour lines</string>
-    <string name="turn_on_nautical_depth_description">Turn on to view nautical depth data on the map.</string>
-    <string name="weather_plugin_description">Weather plugin provides\n- 5 layers…  a 24h hour forecast &amp; 7 day forecast every 3 hours\n- location-based widgets\n- temperature / pressure isolines\nWeather data is provided by GFS https://www.ncei.noaa.gov/products/weather-climate-models/global-forecast.</string>
+    <string name="depth_contour_lines">Depth contour-lines</string>
+    <string name="turn_on_nautical_depth_description">Shows nautical depth data on the map.</string>
+    <string name="weather_plugin_description">- 5 layers.\n- Daily and 7-day forecast updated every 3 hours.\n- location-based widgets\n- temperature / pressure isolines\nWeather data provided by GFS https://www.ncei.noaa.gov/products/weather-climate-models/global-forecast</string>
```

Comment:
No: a daily forecast is different from a 24h forecast.

Reply:
That distinction isn't made in English.

day
noun

1.  each of the twenty-four-hour periods, reckoned from one midnight to the next, into which a week, month, or year is divided, and corresponding to a rotation of the earth on its axis

Example: "they only met a few days ago"